content/en/dynamic_instrumentation/how-it-works.md

```diff
@@ -44,17 +40,69 @@ Dynamic Instrumentation metric probes support the following metric types:
 - [**Gauge**][2]: Generates a gauge based on the last value of a variable. This metric requires a [metric expression](#expression-language).
 - [**Histogram**][3]: Generates a statistical distribution of a variable. This metric requires a [metric expression](#expression-language).
 
+### Span probes
+
+A *span probe* emits a span when a method is executed. You can use a *span probe* as a more efficient alternative to [creating new spans with Custom Instrumentation][4], as it does not require making code changes and redeploying your software. If the method throws an exception, the details of the exception will be associated with the newly created span's `error` tag.
```

Comment:
This is the exact same as on the other page. For log and metric probes we provide additional information compared to the landing page. Is there anything extra we should add about span (tag) probes?

Reply:
I'm not sure - now that we have more probe types, I'm wondering if having this duplication still makes sense, and if we should consider dropping the "How It Works" page entirely?

Reply:
Then we should move any extra information given here to the landing page. If we don't have any extra info for the span probes we could just drop these sections as well. We also definitely need to keep the expression language docs.

Reply:
I've done this refactoring and moved the expression language docs to a different page. WDYT?